scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -184,11 +185,17 @@ def write_tie_out_vcf(vds, vcf_output_path):
     parser.add_argument('--ancestry_input_path', type=str, help='Input ancestry file path', required=True)
     parser.add_argument('--vds_input_path', type=str, help='Input VDS path', default="@VDS_INPUT_PATH@")
     parser.add_argument('--sites_only_output_path', type=str, help='Output sites-only VCF path',
-                        default="@SITES_ONLY_VCF_OUTPUT_PATH@")
+                        default="@SITES_ONLY_VCF_OUTPUT_PATH@"),
+    parser.add_argument('--temp_path', type=str, help='Path to temporary directory', default="@TEMP_DIR@",
+                        required=True)
```

Comment:
having a combination of `required=True` and a `default` seems contradictory

Reply:
Thanks, resolved.